sycl/source/detail/xpti_registry.hpp

```diff
@@ -50,11 +59,13 @@ class XPTIRegistry {
     for (const auto &StreamName : MActiveStreams) {
       xptiFinalize(StreamName.c_str());
     }
+    xptiFrameworkFinalize();
```

Comment:
We always initialize the framework once (it just can be done from multiple threads), but deinitialization is guaranteed to be called exactly once, and from a single thread, because this is a global destructor.